modules/client/src/main/java/org/apache/ignite/internal/client/tx/ClientTransaction.java

```diff
@@ -134,6 +153,23 @@ public static ClientTransaction get(Transaction tx) {
                     + "'. Use IgniteClient.transactions() to start transactions.");
         }
 
-        return (ClientTransaction) tx;
+        ClientTransaction clientTx = (ClientTransaction) tx;
+
+        var state = clientTx.state.get();
```

Comment:
using `var` in the context is forbidden by our codestyle guid

Reply:
Fixed.